package.json

```diff
@@ -37,7 +37,8 @@
   "private": true,
   "repository": "immutable/ts-immutable-sdk.git",
   "resolutions": {
-    "@openzeppelin/contracts": "3.4.2-solc-0.7"
+    "@openzeppelin/contracts": "3.4.2-solc-0.7",
```

Comment:
Should we remove this ?

Reply:
I think we should be able to remove this.